region/wreckedship/main/Bowling Alley.json

```diff
@@ -1318,6 +1319,52 @@
       ],
       "clearsObstacles": ["A"]
     },
+    {
+      "link": [6, 1],
+      "name": "Grapple Block, Crystal Flash, X-Ray Climb",
+      "requires": [
+        {"notable": "Grapple Block Crystal Flash"},
+        "canUseGrapple",
+        "canJumpIntoRespawningBlock",
+        "canTrickyJump",
+        {"or": [
+          "HiJump",
+          {"and": [
+            "canTrickySpringBallJump",
+            "canInsaneJump"
+          ]},
+          {"and": [
+            "canTrickySpringBallJump",
+            {"thornHits": 1}
+          ]},
```

Comment:
With trickyjump, you shouldn't be taking a spike hit. Its pretty safe with a turnaround spinjump.
An extra tech to consider adding would be `canBePatient` since it would be slower to reattempt with springball jumps.

> "spikeHits": 1